website/docs/intro/use-cases.mdx

```diff
@@ -3,37 +3,58 @@ layout: "intro"
+
+In addition to configuring your networks, servers, and databases, you can also use Terraform to manage other resources like containers, machine images, web services, and observability services.
+
+### Resources
+
+- Get started with Terraform on [AWS](/terraform/tutorials/aws-get-started), [Azure](/terraform/tutorials/azure-get-started), [Google Cloud](/terraform/tutorials/gcp-get-started), or [Docker](/terraform/tutorials/docker-get-started).
+- Read HashiCorp's Well-Architected Framework to learn how to [codify your infrastructure with Terraform](/operational-excellence/operational-excellence-manage-your-infrastructure-components-with-terraform).
+
+## Multi-cloud deployment
+
 Provisioning infrastructure across multiple clouds increases fault-tolerance, allowing for more graceful recovery from cloud provider outages. However, multi-cloud deployments add complexity because each provider has its own interfaces, tools, and workflows. Terraform lets you use the same workflow to manage multiple providers and handle cross-cloud dependencies. This simplifies management and orchestration for large-scale, multi-cloud infrastructures.
 
 ### Resources
 
+- Learn how to [standardize artifacts across multiple cloud providers] with Terraform and Packer.
 - Try our [Deploy Federated Multi-Cloud Kubernetes Clusters](/terraform/tutorials/networking/multicloud-kubernetes) tutorial to provision Kubernetes clusters in both Azure and AWS environments, configure Consul federation with mesh gateways across the two clusters, and deploy microservices across the two clusters to verify federation.
 - Browse the [Terraform Registry](https://registry.terraform.io/browse/providers) to find thousands of publicly available providers.
 
+## Multi-environment deployments
+
+You may have staging or QA environments that you use to test new applications before releasing them in production. As the production environment grows larger and more complex, it can be increasingly difficult to maintain an up-to-date environment for each stage of the development process. Terraform Stacks let you split your Terraform configuration into components and then deploy and manage those components across multiple environments. You can manage the lifecycle of each deployment separately, making it easy to validate changes throughout your deployments.
+
+### Resources
+
+- Read the [Deploy a Stack with HCP Terraform](/terraform/tutorials/cloud/stacks-deploy) tutorial to get hands-on experience. You will create a Terraform Stack that deploys an AWS Lambda function across a development, test, and production environment.
```

Comment:
Intro sentence please! Also would it be too much to add the stacks docs too? There is a page about stacks use cases that feels very on-brand with the rest of the article: https://developer.hashicorp.com/terraform/language/stacks/use-cases